README.md

```diff
@@ -326,18 +326,18 @@ All the below functions return a Promise which gets resolved with the result.
 
 This library provides functions for working with Data Unions. To get a DataUnion instance, call `client.getDataUnion(address)`. To deploy a new DataUnion, call `deployDataUnion(options)`
 
-TODO: All `options`-parameters should be documented (see TypeScript interfaces for the definitions)
+See `/docs` folder in this repo for DU `options` parameters.
 
 These DataUnion-specific options are used from `StreamrClient` options:
 | Property                            | Default                                                | Description                                                                                                      |
 | :---------------------------------- | :----------------------------------------------------- | :--------------------------------------------------------------------------------------------------------------- |
-| tokenAddress                        | 0x0Cf0Ee637<br>88A0849fE52<br>97F3407f701<br>E122cC023 | Token used by the DU                                                                                             |
```

Comment:
Users won't typically set any of these parameters. Now when we have the generated docs, this section is maybe redundant. If we want to keep it, we could maybe create a separate page about these corner case details so that the overall usage is easily understandable.

Reply:
This is true, I'll remove them. I would like to have a reference page somewhere.. I'll add a linear ticket for that.

Reply:
https://github.com/streamr-dev/streamr-client-javascript/pull/219/commits/3455171e35fcd63e97c3ae3f3cbf9f64f513f7a2